webapp/src/app/api/pull-request/[projectName]/route.ts

```diff
@@ -38,7 +38,7 @@ export async function POST(
   if (syncLock.get(repoPath) === true) {
     return NextResponse.json(
       {
-        message: `Another Request in progress for the same repository: ${serverProjectConfig.owner}/${serverProjectConfig.repo}`,
+        message: `Another Request in progress for project: ${projectName} or a project that share same git repository`,
```

Comment:
Good catch with the risk of multiple Lyra projects using the same local clone. As a future improvement, we should reject such configurations when starting Lyra.